packages/blade/src/components/Button/Button/Button.tsx

```diff
@@ -29,6 +29,8 @@ type ButtonCommonProps = {
   rel?: BaseButtonProps['rel'];
   variant?: 'primary' | 'secondary' | 'tertiary';
   size?: 'xsmall' | 'small' | 'medium' | 'large';
+  intent?: 'positive' | 'negative' | 'notice';
+  contrast?: 'low' | 'high';
```

Comment:
I don't think so we'll need contrast prop. We can achieve the same result by passing `variant="secondary"` and `intent="negative"` right?

Reply:
@saurabhdaware  If `variant` and `intent` both are defined, `intent` takes priority which makes `variant` ineffective/redundant.

Reply:
> Can we change this? so variant will only decide if the button is filled, outlined etc. But intent will decide the color. And when no intent is passed, the color will be primary blue that we have right now 🤔

@saurabhdaware No. variant secondary and low contrast have different designs.

Reply:
I think that contrast was decided for Alert component earlier. In our case of Button, we don't really need contrast. We need our intent color to support different variants of Button 🤔 

In the context of Button, the contrast might have different meaning (e.g. I would expect blue button to be white button on passing contrast the way we do in text component)

Can discuss this with the team once and take a call.

Reply:
Update: we decided to set contrast `high` by default and expose only `intent`.